### PR TITLE
feat(llmq): Introduction of llmq_devnet_platform + update llmqPlatform from args (devnets only)

### PR DESCRIPTION
**Pull request by ogabrielides**
## Issue being fixed or feature implemented

Currently, by default in devnet, the LLMQ set for Platform is `LLMQ_100_67`.
Obviously this is too big for usual devnets, therefore the new `LLMQ_DEVNET_PLATFORM` is created (size 12, threshold 67%).

`LLMQ_100_67` is still the default one: added possibility to overwrite it by passing argument `-llmqplatform` (devnets only)

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Comment by ogabrielides** [comment]
@thephez Do we need release notes for the extra argument `-llmqplatform` ?

**Comment by thephez** [comment]
> @thephez Do we need release notes for the extra argument `-llmqplatform` ?

If @PastaPastaPasta thinks we do :slightly_smiling_face:

**Comment by UdjinM6** [comment]
>Currently, by default in devnet, the LLMQ set for Platform is LLMQ_100_67.
Obviously this is too big for usual devnets

why 
> LLMQ_100_67 is still the default one

then?

**Comment by ogabrielides** [comment]
> >Currently, by default in devnet, the LLMQ set for Platform is LLMQ_100_67.
> 
> Obviously this is too big for usual devnets
> 
> 
> 
> why 
> 
> > LLMQ_100_67 is still the default one
> 
> 
> 
> then?

Similarly to llmqChainlock, llmqInstantSend, llmqInstantSendDIP0024: by default the larger LLMQs LLMQ_50_60, LLMQ_60_75 are set but they can be overwritten with -llmqchainlock, -llmqinstantsenddip0024 for smaller devnets.

**Review by UdjinM6 — DISMISSED** [review]
hmm.. ok, makes sense (for this PR) I guess. But we should probably consider changing devnet defaults to more realistic ones (and drop most likely useless/unused llmqs like the ones with 400 members).

utACK

**Comment by ogabrielides** [comment]
@UdjinM6 Sometimes we still need to deploy large devnets and rest stuff with conditions close to Testnet 🤷🏼‍♂️

**Comment by UdjinM6** [comment]
> @UdjinM6 Sometimes we still need to deploy large devnets and rest stuff with conditions close to Testnet 🤷🏼‍♂️

hmm..ok.

File: src/llmq/params.h [hunk 1]
```diff
@@ -284,6 +285,31 @@ static constexpr std::array<LLMQParams, 12> available_llmqs = {
...
+     * This quorum is only used for testing on devnets
+     *
+     */
+    LLMQParams{
+        .type = LLMQType::LLMQ_DEVNET_PLATFORM,
+        .name = "llmq_devnet_platform",
+        .useRotation = false,
+        .size = 12,
+        .minSize = 7,
+        .threshold = 8,
```
Comment (thephez) [inline src/llmq/params.h:299]: Threshold can't exceed minsize can it? :thinking:

**Review by thephez — CHANGES_REQUESTED** [review]
Isn't this an issue?

**Comment by ogabrielides** [comment]
@thephez good catch!

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by thephez — APPROVED** [review]
Looks better :+1: 
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Pull request merged**